<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtCyCells</class>
 <widget class="QDialog" name="QtCyCells">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CyCells</string>
  </property>
  <widget class="QPushButton" name="pbDefineFromFile">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Define From File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbInitFromFile">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Initialize From File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbRun">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSetSeed">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Seed</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbWriteHistory">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>40</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Write History</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbWriteSnapshot">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Write Snapshot</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbTimestep">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>610</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Timestep</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbView">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>610</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>View</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDisplayUpdate">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>610</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Display Update</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMoleculeTypes">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Molecule Types: </string>
   </property>
  </widget>
  <widget class="QLabel" name="lblGeometry">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Geometry</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCellTypes">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cell Types	</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDuration">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>610</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Duration</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMoleculeTypes">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>51</width>
     <height>29</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtCellTypes">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>51</width>
     <height>29</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtGeometry">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>231</width>
     <height>45</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTimeElapsed">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>581</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Elapsed Time</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="gfxView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>601</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
